src/main_app/db/db_Settings.py

```
@@ -0,0 +1,118 @@
+            return None
+        return self._parse_value(rows[0]["value"], rows[0]["value_type"])
+
+    def create_setting(self, key: str, title: str, value_type: str, value: Any) -> bool:
+        """Create a new setting."""
+        str_val = self._serialize_value(value, value_type)
+        try:
+            self.db.execute_query_safe(
+                "INSERT IGNORE INTO `settings` (`key`, `title`, `value_type`, `value`) VALUES (%s, %s, %s, %s)",
+                (key, title, value_type, str_val)
+            )
+            return True
```

Ruff (0.15.2)

[warning] Consider moving this statement to an `else` block

(TRY300)